### PR TITLE
Clize wet-bulb-temperature-integral

### PR DESCRIPTION
**Pull request by bayliffe**
Adds a function to load data such that it is no longer lazy.

Testing:
 - [x] Ran tests and they passed OK
 - [x] Added new tests for the new feature(s)

### CODE REVIEW

File: lib/improver/utilities/cli_utilities.py [hunk 1]
```diff
@@ -93,3 +93,17 @@ def radius_or_radii_and_lead(radius=None, radii_by_lead_time=None):
...
+    and not accessed using lazy data.
+
+    Args:
+        cubes (list of iris.cube.Cube or iris.cube.Cube):
+            A list of cubes to be made non-lazy.
+    """
+    if not isinstance(cubes, list):
+        cubes = [cubes]
+    for cube in cubes:
+        cube.data
```
Comment (benfitzpatrick) [inline lib/improver/utilities/cli_utilities.py]: Codacy Issue found: [Statement seems to have no effect](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4683231)

**Comment by benfitzpatrick** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4683231&bid=15662481)

File: lib/improver/tests/utilities/test_cli_utilities.py [hunk 2]
```diff
@@ -30,11 +30,18 @@
 # POSSIBILITY OF SUCH DAMAGE.
 """Unit tests for utilities.cli_utilities."""
 
+import os
+import numpy as np
 import unittest
 from unittest.mock import patch, mock_open
+from tempfile import mkdtemp
```
Comment (benfitzpatrick) [inline lib/improver/tests/utilities/test_cli_utilities.py]: Codacy Issue found: [standard import "from tempfile import mkdtemp" should be placed before "import numpy as np"](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4683231)

File: lib/improver/cli/wet_bulb_temperature_integral.py [hunk 3]
```diff
@@ -76,10 +51,11 @@ def process(wet_bulb_temperature):
         iris.cube.Cube:
             Processed Cube of wet bulb integrals.
     """
-    result = WetBulbTemperatureIntegral().process(
-        wet_bulb_temperature)
-    return result
+    from improver.psychrometric_calculations.psychrometric_calculations \
+        import WetBulbTemperatureIntegral
+    from improver.utilities.cli_utilities import make_cubes_non_lazy
```
Comment (arh89) [inline lib/improver/cli/wet_bulb_temperature_integral.py]: Eek! Ideally I'd like to scrap the whole of `improver.utilities.cli_utilities`.
I think the plugin itself should need to know whether or not it needs non-lazy data - could we move the touching of the data to there, please?  (Do we even need it?)

Comment (arh89) [reply lib/improver/cli/wet_bulb_temperature_integral.py]: (Aside from that I'm happy with this PR)

**Pull request closed without merging**